xmppserver/src/main/java/org/jivesoftware/openfire/session/ClientSessionInfo.java

```diff
@@ -68,6 +72,18 @@ public String getActiveList() {
     public boolean isOfflineFloodStopped() {
         return offlineFloodStopped;
     }
+    
+    /**
```

Comment:
Could you remove the comment? Not really necessary as git will show where/when it was added. Thanks!